server/eclipse-project/src/main/java/us/freeandfair/corla/model/BallotManifestInfo.java

```diff
@@ -218,6 +218,23 @@ public Long sequenceEnd() {
     return my_sequence_end;
   }
 
+  /**
+   * computed value based on other values
```

Comment:
🤔 If I were completely new to this project, it might be helpful to see something like `An imprinted ID, the combination of scanner, batch, and ballot position, is (or can be) printed on ballot cards as they are scanned.`